Sources/SmileID/Classes/SmileID.swift

```diff
@@ -411,13 +411,16 @@ public class SmileID {
     /// - Parameters:
     ///  - config: configuration options for customising biometric verification experience
     ///  - delegate: Callback to be invoked when the Biometric KYC is complete.
+    ///  - onDismiss: Closure to provide custom implementation when biometric kyc screen is dismissed
     public class func biometricKycScreen(
         config: BiometricVerificationConfig,
-        delegate: BiometricKycResultDelegate
+        delegate: BiometricKycResultDelegate,
+        onDismiss: (() -> Void)? = nil
```

Comment:
we don't expose this on android - not opposed to it, but maybe we need to add something like that on android so that the wrappers are in sync

We also use the delegate to send the user cancelled action since you can use the back buttons on android

Reply:
Yeah, basically this is an orchestrated flow. We discussed that we put a constraint on how we want orchestrated flows to be presented. The goal was for orchestrated flows to only be presented as an interstitials and can be dismissed so that users can return back to the original flow. 
This closure is for developers to handle that dismiss if they want to... notice that it's optional. 
The other goal here is to separate navigation actions from job submission events (didSucceed or didError).